core/src/main/java/org/elasticsearch/transport/local/LocalTransport.java

```diff
@@ -223,7 +238,7 @@ public void sendRequest(final DiscoveryNode node, final long requestId, final St
                 throw new NodeNotConnectedException(node, "Node not connected");
             }
 
-            final byte[] data = stream.bytes().toBytes();
+            final byte[] data = bytesStream.bytes().toBytes();
```

Comment:
why use `byteStream` here instead of `stream` since `stream` should delegate it correctly anyway?